client/Makefile

```diff
@@ -80,6 +80,8 @@ test-sdk: ## Run just the sdk tests
 
 .PHONY: regenerate-sdk-cassettes
 regenerate-sdk-cassettes:
+	@echo "Checking if securedrop-proxy has been built locally..."
+	@test -e ../target/debug/securedrop-proxy
```

Comment:
Finding the path to the proxy is a little more complicated, we need to use `cargo metadata` to find the target directory, I'll fix this up. (I wonder if we should have a make target for this!)

Reply:
Or maybe we just `cargo build` to be a little more DWIM.